tests/store/test_store_io_bam.h

```diff
@@ -44,15 +44,15 @@ SEQAN_DEFINE_TEST(test_store_io_bam_read)
   /*
     // Construct name to reference FASTA files.
     char fastaBuffer[1023];
-    strcpy(fastaBuffer, SEQAN_PATH_TO_ROOT());
+    strcpy(fastaBuffer, getAbsolutePath(""));
     strcat(fastaBuffer, "/projects/tests/store/toy.fa");
     // Construct file name to SAM file.
     char samBuffer[1023];
-    strcpy(samBuffer, SEQAN_PATH_TO_ROOT());
+    strcpy(samBuffer, getAbsolutePath(""));
     strcat(samBuffer, "/projects/tests/store/toy.sam");
     // Construct file name to BAM file.
     char bamBuffer[1023];
-    strcpy(bamBuffer, SEQAN_PATH_TO_ROOT());
+    strcpy(bamBuffer, getAbsolutePath(""));
```

Comment:
I don't know :confused:. I regexed the replacement so I accidentally changed unused code. 

There is a ```TODO(holtgrew): Actually check contents.``` at the end so maybe he knows why..